assets/src/edit-story/components/canvas/pagenav/index.js

```diff
@@ -72,6 +73,19 @@ function PageNav({ isNext }) {
     width: PAGE_NAV_BUTTON_WIDTH,
     height: PAGE_NAV_BUTTON_WIDTH,
   };
+
+  if (isRTL) {
+    return (
+      <Wrapper isNext={isNext}>
+        {isNext ? (
+          <LeftArrow {...buttonProps} />
```

Comment:
Ditto: here the naming is even better. I think this is wholly the styling problem in this case.